### PR TITLE
misc: ignore wgpu interpreter file

### PR DESCRIPTION
**Pull request by superlopuh**
The dependency has linting issues, I don't think we care about them

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@ typeCheckingMode = "strict"
     "tests/test_frontend_op_resolver.py",
     "tests/test_frontend_python_code_check.py",
     "tests/dialects/test_memref.py",
+    "xdsl/interpreters/experimental/wgpu.py",
```
Comment (superlopuh) [inline pyproject.toml:12]: eclude means the errors will still be there if you open the file in your IDE, but not otherwise

**Review by math-fehr — APPROVED** [review]
Nice, this would solve most of the remaining errors I have on the pyright bump.
Any reason why we didn't get errors before?

**Comment by math-fehr** [comment]
Actually wait, I don't see them locally anymore, and actually I don't see it in the CI?

**Review by PapyChacal — APPROVED** [review]

**Comment by superlopuh** [comment]
You probably accidentally updated your vscode and are now seeing the pylance errors from 318, I think that's what's happening to me locally and is why Emilien did not see them on his machine

**Pull request merged**